Gruntfile.js

```diff
@@ -36,8 +42,32 @@ module.exports = function (grunt) {
                     }
                 }
             }
+        },
+        uglify: {
+            options: {
+                mangle: false,
+                output: { comments: 'some' },
+                sourceMap: true
+            },
+            target: {
+                files: {
+                    'uber/static/deps/combined.min.js': ['uber/static/deps/combined.js']
+                }
+            }
+        },
+        cssmin: {
+            options: {
+                sourceMap: true
+            },
+            target: {
+                files: {
+                    'uber/static/deps/combined.min.css': ['uber/static/deps/combined.css']
+                }
+            }
         }
     });
     grunt.loadNpmTasks('grunt-bower-concat');
-    grunt.registerTask('default', ['bower_concat']);
+    grunt.loadNpmTasks('grunt-contrib-uglify');
+    grunt.loadNpmTasks('grunt-contrib-cssmin');
+    grunt.registerTask('default', ['bower_concat', 'uglify', 'cssmin']);
```

Comment:
This is a thing we should have been doing for a long time. I figured as long as I was adding `jquery.select-to-autocomplete.js` to `combined.js`, I might as will minify our js and css.